<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arinc665Qt::DecompileMediaSetSettingsPage</class>
 <widget class="QWizardPage" name="Arinc665Qt::DecompileMediaSetSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Settings</string>
  </property>
  <property name="subTitle">
   <string>Select Media Directories</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Arinc665Qt::MediaPathsWidget" name="mediaPaths">
     <property name="title">
      <string>Media Paths</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkFileIntegrity">
     <property name="text">
      <string>Check File Integrity</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Arinc665Qt::MediaPathsWidget</class>
   <extends>QGroupBox</extends>
   <header location="global">arinc665_qt/MediaPathsWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
